src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -22,19 +22,18 @@ To create and connect to a 30-day free CockroachDB {{ site.data.products.dedicat
 1. [Log in](https://cockroachlabs.cloud/) to your CockroachDB {{ site.data.products.cloud }} account.
 1. If there are multiple [organizations](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#organization) in your account, select the correct organization in the top right corner.
 1. On the **Overview** page, click **Create Cluster**.
-1. Select the **Dedicated standard** or **Dedicated advanced** plan.
+1. On the **Select a tier** page, select the **Dedicated standard** or **Dedicated advanced** plan.
```

Comment:
(Another) This is now "Select a plan" https://www.figma.com/file/idjQ4EtJs78TuKTswKCTiW/Cloud-2.0?type=design&node-id=3513-98852&mode=design&t=0gNDZEm9SF3cYn8f-4

Reply:
Fixed